libcloud/compute/providers.py

```diff
@@ -147,6 +147,8 @@
     ('libcloud.compute.drivers.oneandone', 'OneAndOneNodeDriver'),
     Provider.UPCLOUD:
     ('libcloud.compute.drivers.upcloud', 'UpcloudDriver'),
+    Provider.NTTCIS:
```

Comment:
The dimension data key is still in there, do you want to change that to NTT-CIS as well?

Reply:
I was going to remove the Dimension Data piece altogether once I have the nttcis modules up to date with latest API release.